src/PrestaShopBundle/Form/Admin/Improve/International/Localization/ImportLocalizationPackType.php

```diff
@@ -0,0 +1,178 @@
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShopBundle\Form\Admin\Improve\International\Localization;
+
+use PrestaShop\PrestaShop\Core\ConfigurationInterface;
+use PrestaShop\PrestaShop\Core\Localization\Pack\Import\LocalizationPackImportConfig;
+use PrestaShop\PrestaShop\Core\Localization\Pack\Loader\LocalizationPackLoaderInterface;
+use PrestaShopBundle\Form\Admin\Type\SwitchType;
+use PrestaShopBundle\Form\Admin\Type\TranslatorAwareType;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Translation\TranslatorInterface;
+
+/**
+ * Class ImportLocalizationPackType is responsible for building 'Import a localization pack' form
+ * in 'Improve > International > Localization'
+ */
+class ImportLocalizationPackType extends TranslatorAwareType
+{
+    /**
+     * @var LocalizationPackLoaderInterface
+     */
+    private $remoteLocalizationPackLoader;
+
+    /**
+     * @var LocalizationPackLoaderInterface
+     */
+    private $localLocalizationPackLoader;
+
+    /**
+     * @var ConfigurationInterface
+     */
+    private $configuration;
+
+    /**
+     * @param TranslatorInterface $translator
+     * @param array $locales
+     * @param LocalizationPackLoaderInterface $remoteLocalizationPackLoader
+     * @param LocalizationPackLoaderInterface $localLocalizationPackLoader
+     * @param ConfigurationInterface $configuration
+     */
+    public function __construct(
+        TranslatorInterface $translator,
+        array $locales,
+        LocalizationPackLoaderInterface $remoteLocalizationPackLoader,
+        LocalizationPackLoaderInterface $localLocalizationPackLoader,
+        ConfigurationInterface $configuration
+    ) {
+        parent::__construct($translator, $locales);
+
+        $this->remoteLocalizationPackLoader = $remoteLocalizationPackLoader;
+        $this->localLocalizationPackLoader = $localLocalizationPackLoader;
+        $this->configuration = $configuration;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('iso_localization_pack', ChoiceType::class, [
+                'choices' =>  $this->getLocalizationPackChoices(),
+                'attr' => [
+                    'data-toggle' => 'select2',
+                    'data-minimumResultsForSearch' => '7',
```

Comment:
it's invalid to use capital letters here, how about `data-minimum-results-for-search` ?

Reply:
i dont like it either, but i'm not sure if i can use `data-minimum-results-for-search` because it is not  implemented by this PR.

Reply:
ok, we'll change it in another PR /c @eternoendless

Reply:
Both of you are right, you can implement camel case after `data-` and you can use `data-minimum-results-for-search` but agree with @mickaelandrieu it's weird but (again) jquery.select2 does not implement the opts `minimum-results-for-search`, lile ProductFeature.php, ProductPrice.php you must use `data-minimumResultsForSearch` to make it work in the BO.

Reply:
sad story... would you mind to create an issue into prestafony project so I can plan a refactoring at some point.